### ISSUE TITLE
Revisit gravatar aspect of image component on setup/signup

### ISSUE THREAD

**Issue by ErisDS** [TryGhost/Ghost#5797]
During the build out of Zelda, the behaviour of the gravatar tied to the email fields on setup & signup changed, such that the gravatar request isn't fired until the field is de-focused. This doesn't give the same instant effect it had when it was first implemented. 

The change was made in order to reduce the number of requests made, but I think this can be improved to only fire the request only once the email the user has typed is valid. Checking the validation as the user is typing could be debounced as well.

I think there may be room to revisit validations as well in this regard, tying certain operations (like clearing an error on an invalid email) to occur whilst the user is typing. That's probably for another issue though :)
